deploy.sh

```diff
@@ -11,19 +11,16 @@ help() {
 EOH
 }
 deploy_generic() {
-    local environment="${1?No environment passed}"
-
-    if [[ "$environment" = "devel" ]]; then
-        sed "s/{{HARBOR_IP}}/${HARBOR_IP}/"\
-          deploy/devel/auth-patch.yaml.template > deploy/devel/auth-patch.yaml
-    fi
+    
+    sed "s/{{HARBOR_IP}}/${HARBOR_IP}/"\
+      deploy/$environment/auth-patch.yaml.template > deploy/$environment/auth-patch.yaml
```

Comment:
This template file (`auth-path.yaml.template`) only exists in the dever environment, so we still need the `if` block too